internal/output/html/style.html

```diff
@@ -56,101 +61,437 @@
   header {
     display: flex;
     align-items: center;
-    margin-bottom: 20px;
-    position: relative;
-    justify-content: space-evenly;
+    margin-bottom: 50px;
   }
 
   .logo {
-    height: 50px;
-    position: absolute;
-    left: 0;
+    height: 20px;
+    /* position: absolute; */
+    /* left: 0; */
+  }
+
+  header .vl {
+    border-left: 2px solid #fff;
+    height: 25px;
+    margin-left: 20px;
+    margin-right: 20px;
+  }
+
+  header h1 {
+    font-size: 23px;
+  }
+
+  .material-icons {
+    vertical-align: middle;
   }
 
   .vuln-table {
     width: 100%;
-    text-align: center;
+    text-align: left;
     border-radius: 5px;
-    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
     margin-bottom: 40px;
     margin-top: 10px;
+    color: rgba(255, 255, 255, 0.87);
+    border-spacing: 0;
   }
 
   .vuln-table th {
-    border-bottom: 1px solid #ddd;
+    border-bottom: 1px solid rgba(255, 255, 255, 0.12);
+    padding: 16px;
   }
 
-  .vuln-tr {
-    cursor: pointer;
+  .table-tr {
     line-height: 30px;
   }
 
-  .vuln-tr td {
-    border-bottom: 1px solid #eee;
+  .table-tr td {
+    border-bottom: 1px solid rgba(255, 255, 255, 0.12);
     word-break: break-word;
     width: 200px;
+    padding: 16px;
   }
 
-  .vuln-tr:hover {
-    background-color: #eee;
+  .table-tr:hover {
+    background-color: rgba(255, 255, 255, 0.04);
   }
 
   .vuln-id {
     color: #d9534f;
   }
 
-  .vuln-details {
+  .package-details {
     text-align: left;
     padding: 0 50px;
     display: none;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    border-bottom: 1px solid rgba(255, 255, 255, 0.12);
+    max-height: 800px;
+    overflow: overlay;
```

Comment:
issue: [`overlay` is not an official property](https://caniuse.com/css-overflow-overlay) - it technically has a difference to `auto` (which MDN suggests it's an alias for), but ultimately I think we should try to avoid using it even if that requires more work:

```suggestion
    overflow: auto;
```